doc/htmldoc/models/index.rst

```diff
@@ -0,0 +1,173 @@
+.. _modelsmain:
+
+
+Model directory
+===============
+
+
+
+Find a model
```

Comment:
Can this section be eliminated? "Find a model" could go straight to the model selector. The "learn more about" could go as a separate header/paragraph at the bottom of the page.

Reply:
yes, agreed